package dtest.base.exceptions;

/**
 * Thrown when a script calls the $fail function to fail a test.
 */
public class FailException extends RuntimeException {
    public FailException(String message) {
        super(message);
    }
}
